### PR TITLE
Hc review assistant

### PR DESCRIPTION
**Pull request by konarkmodi**
Fixes #??.

Changes proposed in the PR fix/change the following:
- 
- 
- 

To the reviewer: *If you approve this PR, please merge it right away too.*
(Why merging it right away? It makes the PR valuable as soon as it is approved.
The author wrote it with the intention for it to go live, so let's do them the favor as soon as possible :).
Thanks.)

### CODE REVIEW

**Review by tmayer — COMMENTED** [review]
I was wondering whether we want to add a paragraph about the underlying technology that was used to create the assistant? I wouldn't give away too much in here, but just mentioning some things might whet the reader's appetite.

File: _posts/2023-04-18-ai-based-assistant.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+***Submit a test review: Writing reviews will be easier and faster in the future***
+
+HolidayCheck is building an [AI assistant](https://holidaycheck.de/svc/hc-review-assistant/travel-escapes/welcome.html) to make it even easier for holiday makers to write reviews in the future. This is still a beta version to be tested by as many passionate travellers as possible.
+
+Therefore, we are calling on all of you to submit a test review using the HolidayCheck assistant with the beta version. The submitted reviews will not go online but will be used to make the functionality even better. All participants will be entered into a raffle: This is done automatically once you interact with the assistant, and it generates reviews for you.
+
+***Generative AI can help people find the right way to express themselves***
```
[the remark this answers is not in the document]
Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:23]: Calling people for reviews can be the next section with more highlight on the raffle :)

File: _posts/2023-04-18-ai-based-assistant.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+We asked both experts for more information about the HolidayCheck review assistant. Here are their answers:
+
+***Why did you develop your own AI assistant for HolidayCheck?***
+
+We strive to reduce the barrier to entry for users wanting to write a review. One does not need to be an expert in any language to submit a review, nor does one have to spend hours finding the words to use or not use. The users can simply start speaking or typing and the assistant helps to write a proper narrative with just a few words.
```
Comment (TejashreeAher) [inline _posts/2023-04-18-ai-based-assistant.md:31]: barrier-to-entry is one word

Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:31]: "We strive to reduce the barrier-to-entry in writing a review" can be a shorter sentence

Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:31]: One does not need to be a language expert to submit a review, nor does one have to spend hours finding the right words to use

Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:31]: "The assistant comes handy. Just start speaking or typing and the assistant helps to write a proper narrative with just a few words." (to make it more attractive !)

File: _posts/2023-04-18-ai-based-assistant.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+***Is it also possible to upload pictures?***
+
+In the beta phase we are only collecting two types of content: Text and audio. If you wish to submit pictures, they can still be uploaded via our regular system: https://www.holidaycheck.de/wcf
```
Comment (TejashreeAher) [inline _posts/2023-04-18-ai-based-assistant.md:35]: "text" also in small.

File: _posts/2023-04-18-ai-based-assistant.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+We are looking forward to your feedback! You can give your feedback directly either by using the Review Assistant or by clicking the “Feedback” button on the right side of the page.
+
+***How long will it be before I can see the finished feature?***
+
+We are still working on the technology and on the features. We plan to launch it later this year. For more information, please sign up to our newsletter: https://www.holidaycheck.de/newsletter
+
+***Do we risk diminishing the value of genuine reviews vs AI written reviews?***
+
+This is a fair question and one we ought to ask ourselves. However, reviews written by this assistant will still go through all our strict checks and balances. We would not start by publishing them directly, but by carefully moderating them. We must keep in mind that it’s still a review. The users can still use assistants outside of HolidayCheck, so we should accept this and embrace this technology trend – without diluting the brand value and authority.
```
Comment (TejashreeAher) [inline _posts/2023-04-18-ai-based-assistant.md:47]: This is a fair question we ought to ask ourselves. However ...

Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:47]: This is slightly complicated to ask. Do we mean,
"This is a fair question we ought to ask ourselves. Users, even today, can use assistants outside of HolidayCheck to write reviews and we should definitely embrace the technology trend. However, all reviews submitted to HC, including ones generated by this assistant go through all our strict checks and balances, thus not diluting the brand value and authenticity."

File: _posts/2023-04-18-ai-based-assistant.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+***If we publish these reviews, do we mark them as AI assisted?***
+
+Yes, these reviews are labeled and, by the way, go through the same rigorous quality controls and review processes before they are published. Currently the submitted reviews will not go online.
```
[the remark this answers is not in the document]
Comment (TejashreeAher) [reply _posts/2023-04-18-ai-based-assistant.md:51]: "the reviews generated by this tool, whenever published online, will be labeled after going through the same rigorous quality controls and review processes. Currently, the reviews do not go online in the Beta phase." 
would make it more clear ?

**Review by TejashreeAher — CHANGES_REQUESTED** [review]

**Review by TejashreeAher — APPROVED** [review]

**Pull request merged**